app/Http/routes.php

```diff
@@ -288,6 +288,12 @@
     Route::get('organizations', 'AdminController@viewOrganizations');
 
     /**
+     * Admin view and restore deleted events
+     */
+    Route::get('deleted_events', 'AdminController@viewDeletedEvents');
```

Comment:
`admin/cancelled_events`. We will gradually change all admin routes to `admin/*` later in #156